src/main/java/org/ohdsi/authenticator/service/AuthenticatorImpl.java

```diff
@@ -47,14 +46,13 @@ private void postConstruct() throws ClassNotFoundException, NoSuchMethodExceptio
 
     @Override
     public UserInfo authenticate(String method, Credentials request) {
-
-        var authService = getForMethod(method);
+        AuthService authService = getForMethod(method);
```

Comment:
Let's be consistent and not move the formatting back and forwards: by default (ctrl+alt+L) Idea inserts an empty line at the beginning of a method, so why do you remove that?

Reply:
done